platform/viewer/cypress/integration/pwa/OHIFExtensionHTML.spec.js

```diff
@@ -22,4 +22,23 @@ describe('OHIF HTML Extension', () => {
       'Imaging Measurement Report'
     );
   });
+
+  it.only('checks if the HTML viewport has been set to active by interaction', () => {
```

Comment:
`it.only` -- haven't seen that before. How many flavors of this are there?

Reply:
Oh, good catch! `.only` runs only this specific test. I'm gonna remove it, my mistake!